interop-cats/jvm/src/test/scala/zio/interop/CatsZManagedSyntaxSpec.scala

```diff
@@ -105,7 +108,7 @@ class CatsZManagedSyntaxSpec extends Specification with AroundTimeout with Defau
 
     val testCase = {
       val managed: ZManaged[Any, Throwable, Unit] = res(1).toManaged
-      managed.use(_ => ZIO.unit)
+      managed.use[Any, Throwable, Unit](_ => ZIO.unit)
```

Comment:
rip inference 😢

Reply:
It infers correctly, but `-XLint` makes it fail
```
[error] /Users/pierangelo.cecchetto/Documents/projects/myprojects/interop-cats/interop-cats/jvm/src/test/scala/zio/interop/CatsZManagedSyntaxSpec.scala:111:15: a type was inferred to be `Any`; this may indicate a programming error.
[error]       managed.use(_ => ZIO.unit)
[error]               ^
[error] one error found
[error] (interopCatsJVM / Test / compileIncremental) Compilation failed
```

Reply:
You can do `.use_(ZIO.unit)`. Does that infer better?

Reply:
Seems to compile well with `use_`. I'll push that.